scout/server/blueprints/omics_variants/controllers.py

```diff
@@ -48,3 +58,62 @@ def decorate_omics_variant(store, institute_obj, case_obj, omics_variant_obj):
     update_variant_case_panels(case_obj, omics_variant_obj)
 
     return omics_variant_obj
+
+
+def download_omics_variants(case_obj: dict, variant_objs: CursorType):
+    """Download omics variants in a csv file."""
+
+    def generate(header, lines):
+        yield header + "\n"
+        for line in lines:
+            yield line + "\n"
+
+    DOCUMENT_HEADER = [
+        "Gene",
+        "Gene annotation",
+        "category",
+        "sub-category",
+        "Potential impact",
+        "Value",
```

Comment:
We will likely want to split this one into `delta_psi` and `l2fc`, for clarity and ease of handling - even if the `sub_category` column does show what gives.

Reply:
> We will likely want to split this one into `delta_psi` and `l2fc`, for clarity and ease of handling - even if the `sub_category` column does show what gives.

Ah OK, good idea!

Reply:
Fixed like this now:

<img width="469" alt="image" src="https://github.com/user-attachments/assets/cd6328e8-50b5-4ec1-b467-2d34a2916fcd">